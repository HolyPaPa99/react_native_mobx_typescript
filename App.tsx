/**
 * Sample React Native App
 * https://github.com/facebook/react-native
 *
 * @format
 * @flow
 */

import React from 'react';
import {Provider} from 'mobx-react';
import 'react-native-gesture-handler';
import {createAppContainer} from 'react-navigation';
import {createStackNavigator} from 'react-navigation-stack';
import stores from '@/models';
import {RouteConfigs, NavigatorConfig} from '@/navigation/myNavigator';
import {createBottomTabNavigator} from 'react-navigation-tabs';

const navigator = createBottomTabNavigator(RouteConfigs);
const AppContainer = createAppContainer(navigator);

const App = () => {
  return (
    <Provider {...stores}>
      <AppContainer />
    </Provider>
  );
};

export default App;
