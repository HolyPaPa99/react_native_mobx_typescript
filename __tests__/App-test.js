/**
 * @format
 */

import 'react-native';
import React from 'react';
import Hello from '@/screens/hello'
import App from '../App'
// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

// it('renders correctly', () => {
//   renderer.create(<App />);
// });

test('renders correctly', () => {
  const tree = renderer.create(<Hello />).toJSON();
  expect(tree).toMatchSnapshot();
});
