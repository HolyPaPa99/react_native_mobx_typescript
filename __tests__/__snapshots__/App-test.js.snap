// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<RCTSafeAreaView
  emulateUnlessSupported={true}
>
  <View>
    <Text>
      hello
    </Text>
  </View>
</RCTSafeAreaView>
`;
